src/components/LayoutHeader/Header.js

```diff
@@ -13,6 +13,7 @@ import React from 'react';
 import {colors, fonts, media} from 'theme';
 import {version} from 'site-constants';
 import ExternalLinkSvg from 'templates/components/ExternalLinkSvg';
+import DocSearch from 'components/DocSearch';
```

Comment:
Organizational nit: This new `DocSearch` component can just live within the `src/components/LayoutHeader` folder since it's only referenced there.

Particularly since there are styles defined on the `DocSearch` component (eg `flex: '0 0 auto'`) that are dependent on its parent.